index.md

```diff
@@ -10,7 +10,7 @@ We get together once a month to talk about anything related to the Python progra
 
 Anyone who is interested in Python is welcome, whatever your level of knowledge (we're very happy to help with questions from those just learning the language), and for whatever purpose you use it.
 
-We meet on the first Tuesday of each month, normally at the Raspberry Pi Foundation offices, 30 Station Road, Cambridge, CB1 2JH. Meetings are from 7pm until 9pm, and afterwards some of us go on to the pub.
+We meet on the first Tuesday of each month, normally at the Raspberry Pi Foundation offices, 37 Hills Rd, Cambridge, CB2 1NF. Meetings are from 7pm until 9pm, and afterwards some of us go on to the pub.
```

Comment:
Should be `CB2 1NT`

Reply:
I copied the address from https://www.meetup.com/CamPUG/ so I guess that needs fixing too :wink: